src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -410,11 +410,14 @@ about preferred names of coordinate systems, such as ACPC).
 
 ### Allowed 2D coordinate systems
 
-If electrodes are localized in 2D space (only x and y are specified and z is
-n/a), then the positions in this file must correspond to the locations expressed
-in pixels on the photo/drawing/rendering of the electrodes on the brain. In this
-case, coordinates must be (row,column) pairs, with (0,0) corresponding to the
-upper left pixel and (N,0) corresponding to the lower left pixel.
+If electrodes are localized in 2D space (only x and y are specified and z is `"n/a"`),
+then the positions in this file MUST correspond to the locations expressed
+in pixels on the photo/drawing/rendering of the electrodes on the brain.
+In this case, `iEEGCoordinateSystem` MUST be defined as `"Pixels"`,
+and `iEEGCoordinateUnits` MUST be defined as `"pixels"`
+(note the capitalization).
```

Comment:
This is kind of a tiny thing, but I'd either switch the order or say "note the difference in capitalization".

Reply:
good point, thanks.

```suggestion
In this case, `iEEGCoordinateSystem` MUST be defined as `"Pixels"`,
and `iEEGCoordinateUnits` MUST be defined as `"pixels"`
(note the difference in capitalization).
```